linkage-monitor/src/test/resources/testproject/testsubproject/build/pom.xml

```diff
@@ -0,0 +1,14 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
This test is added LinkageMonitorTest.testFindLocalArtifacts ensures this file is not picked up.